src/middlewares/register.middleware.ts

```diff
@@ -0,0 +1,28 @@
+import * as Koa from 'koa'
+import * as JWT from 'jsonwebtoken'
+import { hashSync } from 'bcrypt'
+import User from '../models/user.entity'
+require('dotenv').config()
+
+export default async function(ctx: Koa.Context, next: Koa.Next) {
+    try {
+        const user: User = {
+            name: ctx.request.body.name,
+            secondname: ctx.request.body.secondname,
+            email: ctx.request.body.email,
+            password: hashSync(ctx.request.body.password, 10),
```

Comment:
Data in JWT token isn't secure, so it is better not to store there something like password, even encrypted.